akka-stream-tests/src/test/scala/akka/stream/impl/TimeoutsSpec.scala

```diff
@@ -130,6 +130,117 @@ class TimeoutsSpec extends AkkaSpec {
 
   }
 
+  "BackpressureTimeout" must {
+
+    "pass through elements unmodified" in assertAllStagesStopped {
+      Await.result(Source(1 to 100).backpressureTimeout(500.millis).grouped(200).runWith(Sink.head), 3.seconds) should === (1 to 100)
+    }
+
+    "succeed if subscriber demand arrives" in assertAllStagesStopped {
+      val subscriber = TestSubscriber.probe[Int]()
+
+      Source(1 to 4)
+        .backpressureTimeout(500.millis)
```

Comment:
this might be too tight on our build machines and cause spurious timeout errors.